src/app/private/scripts/factories/dashboard/rendererService.js

```diff
@@ -0,0 +1,141 @@
+'use strict'
+
+angular.module('playerApp')
+  .factory('rendererService', ['$filter', 'config', '$timeout', 'toasterService',
+    'uuid4',
+    function ($filter, config, $timeout, toasterService, uuid4) {
+      /**
+       * @method Render
+       * @desc - render line chart
+       * @param   {object}  data
+       * @param   {string}  dashboardType
+       */
+      function Render (data, dashboardType) {
```

Comment:
Please split passing of data into the service and rendering of the chart into separate functions.